pyTANSPEC/reduce_specTANSPEC.py

```diff
@@ -225,7 +225,10 @@ def biweight_fits_files(files_list,
                 loglist.write('\n'+files)
                 loglist.close()
                 data = fits.getdata(files, ext=0)
-                var = fits.getdata(files, ext=1)
+                try:
+                    var = fits.getdata(files, ext=1)
+                except:
```

Comment:
The except should not be generic. Make the Error that you want to catch explicit.

Reply:
But for the users who use the quick looks frames, do not have variance. They have only one extension. That's why I added this.

Reply:
Okay, but catch only that particular error.

Reply:
This is fixed.